src/cript/nodes/primary_nodes/collection.py

```diff
@@ -59,10 +59,10 @@ class JsonAttributes(PrimaryBaseNode.JsonAttributes):
         # TODO add proper typing in future, using Any for now to avoid circular import error
         member: List[User] = field(default_factory=list)
         admin: List[User] = field(default_factory=list)
-        experiment: Optional[List[Any]] = None
-        inventory: Optional[List[Any]] = None
+        experiment: List[Any] = field(default_factory=list)
```

Comment:
oh I see, so by default you gave it a list in case not provided by API

Reply:
Yes, exactly